andela/wg-tomatoes
# #154240674 Fix Python Style Violations

#### What does this PR do?
Fix style violations in the **_wger_** codebase and allow for exclusions where necessary

#### Description of Task to be completed?
Identify all style violations.
Review those that require fixing and those that require exclusion(and include them in the .flake8 ini file)
Conform style violations that require fixing to a standard.(PEP8 in this case)

#### How should this be manually tested?
Install flake8 via PyPI
In a terminal, in the root of the project folder,  run the command `flake8 .`
The command should exit with no output showing that no PEP8 style violations were found.

#### What are the relevant pivotal tracker stories?
154240674 Python coding style warnings/errors
